skimage/__init__.py

```diff
@@ -159,3 +160,7 @@ def _raise_build_error(e):
 
 
 del warnings, functools, osp, imp, sys
+
+
+def lookfor(search_text):
+    return np.lookfor(search_text, sys.modules[__name__])
```

Comment:
I made a mistake in my first comment here.  It should have been
```python
 sys.modules[__name__]
```
as you originally had within the function rather than
```python
sys.modules['name']
```
The invalid key is causing the current CI failures